### PR TITLE
metering-read-fixed

### PR DESCRIPTION
**Pull request by gopal-mahajan**

### CODE REVIEW

**Comment by kailash** [comment]
ok to test

**Comment by kailash** [comment]
@gopal-mahajan coverage is decreased by -2% (90 to 88%), please fix this.

**Comment by karun-singh** [comment]
retest this please

File: src/main/java/iudx/resource/server/metering/util/QueryBuilder.java [hunk 1]
```diff
@@ -87,37 +91,105 @@ public JsonObject buildReadingQuery(JsonObject request) {
...
         query =
             new StringBuilder(
-                PROVIDERID_TIME_INTERVAL_COUNT_QUERY
+                PROVIDERID_TIME_INTERVAL_READ_QUERY
                     .replace("$0", databaseTableName)
                     .replace("$1", Long.toString(fromTime))
                     .replace("$2", Long.toString(toTime))
                     .replace("$3", providerID));
-      } else {
+      else
```
Comment (kailash) [inline src/main/java/iudx/resource/server/metering/util/QueryBuilder.java:102]: missing `{`  `}` braces, better to write if else inside braces.

**Review by kailash — CHANGES_REQUESTED** [review]

File: src/main/java/iudx/resource/server/metering/util/QueryBuilder.java (hunk 1 above)
Comment (gopal-mahajan) [reply src/main/java/iudx/resource/server/metering/util/QueryBuilder.java]: done

**Pull request merged**